jenkins-x.yml

```diff
@@ -56,6 +56,15 @@ pipelineConfig:
                 - --cache-repo=gcr.io/jenkinsxio/cache
                 - --cache=true
                 - --cache-dir=/workspace
+            - name: build-and-push-maven-graalvm
+              command: /kaniko/executor
+              args:
+                - --dockerfile=/workspace/source/builder-maven-graalvm/Dockerfile
+                - --destination=gcr.io/jenkinsxio/builder-maven-graalvm:${inputs.params.version}
+                - --context=/workspace/source
+                - --cache-repo=gcr.io/jenkinsxio/cache
+                - --cache=true
+                - --cache-dir=/workspace
```

Comment:
@carlossg There also needs to be entries for the pullRequest & release pipelines

Reply:
ah sorry missed the other one. Fixed now